gpu/gosl/diff/diff.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2022 The Go Authors. All rights reserved.
```

Comment:
Where did you get this code? You should indicate that. Also, is this duplicative with our `difflib`?

Reply:
unused., removed